sdk/storage/azblob/common.go

```diff
@@ -24,6 +24,7 @@ func NewSharedKeyCredential(accountName, accountKey string) (*SharedKeyCredentia
 // NOTE: Changing any SAS-related field requires computing a new SAS signature.
 type URLParts = sas.URLParts
 
+// TODO : Can we delete below? Currently unused.
```

Comment:
I thought we wanted this in `azblob` as part of SAS support.  Also, we export `URLParts` above.  So it would seem weird to export it and not `ParseURL()` from the same package.

Reply:
It just seems like it's duplicate code, clients (container, pageblob, etc.) are using the parser (and URLParts) defined here https://github.com/Azure/azure-sdk-for-go/blob/main/sdk/storage/azblob/blob/utils.go#L77
so this seems safe to remove along with URLParts from this common file, unless we want to use this instead of the one defined in utils @jhendrixMSFT

Reply:
The original thinking was that the contents of the `azblob` package should address the 95% use-case (i.e. you should only have to import `azblob` and no other sub-packages), and SAS support was determined to be part of that 95%.

Reply:
@jhendrixMSFT I see, we can keep this then